src/WebJobs.Extensions/Extensions/Core/Bindings/ErrorTriggerAttributeBindingProvider.cs

```diff
@@ -1,170 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Globalization;
-using System.Linq;
-using System.Reflection;
-using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.Extensions.Bindings;
-using Microsoft.Azure.WebJobs.Extensions.Core.Listener;
-using Microsoft.Azure.WebJobs.Host;
-using Microsoft.Azure.WebJobs.Host.Bindings;
-using Microsoft.Azure.WebJobs.Host.Listeners;
-using Microsoft.Azure.WebJobs.Host.Protocols;
-using Microsoft.Azure.WebJobs.Host.Triggers;
-
-namespace Microsoft.Azure.WebJobs.Extensions.Core
-{
-    internal class ErrorTriggerAttributeBindingProvider : ITriggerBindingProvider
```

Comment:
I'm somewhat OK with temporarily removing this as long as we prioritize adding it back soon by addressing https://github.com/Azure/azure-webjobs-sdk-extensions/issues/342. I don't want this feature to become a casualty.